src/mlpack/methods/bayesian_linear_regression/bayesian_linear_regression_main.cpp

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file bayesian_linear_regression_main.cpp
+ * @author Clement Mercier
+ *
+ * Executable for BayesianLinearRegression.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+
+#include "bayesian_linear_regression.hpp"
+
+using namespace arma;
+using namespace std;
+using namespace mlpack;
+using namespace mlpack::regression;
+using namespace mlpack::util;
+
+PROGRAM_INFO("BayesianLinearRegression",
+    // Short description.
+    "An implementation of the bayesian linear regression. This can train a "
+    "Bayesian linear regression model and use that model or a pre-trained "
+    "model to output regression predictions for a test set.",
+    // Long description.
+    "An implementation of the bayesian linear regression, also known"
+    "as the Bayesian linear regression.\n "
```

Comment:
This seems redundant, do you think that `An implementation of the Bayesian linear regression.` is sufficient here?  My comment is that the same term is on both sides of the `also known as`. :)